### DISCUSSION TITLE
Linux support? Trying Lutris but no success

### DISCUSSION THREAD

**Discussion by marcelo-js** [C1rdec/PoeLurker#406] [Q&A] [answered]
I tried to install PoE Lurker using Lutris since there's only an .exe available in the releases:
- Pressed + sign on Lutris
- Install a Windows game from media
- Selected the PoeLurkerSetup.exe

Then I get a pop up window with:

```
Installation has failed
Please re-run this installer as a normal user instead of "Run as Administrator"
```

While looking around, I saw that there might be a way to compile this using dotnet but I haven't figured it out yet.
Does someone have any pointers or is it too much extra work to provide a Linux compatible executable?
I'm on Kubuntu if that helps.

Thanks!

**Comment by C1rdec** [comment, marked as answer]
You wont be able to make this work since this is based out of WPF (Windows Presentation Framework).

In 2.0 I'm looking at new .Net UI Framework to support Linux

**Comment by dyfrgi** [comment]
Is this something you are still pursuing or have already implemented? I can't find anything in issues etc about Linux support today.

**Comment by YaronFainstein-Vald** [comment]
Any updates on this ? thinking about ditching the windoze (sinking) ship, and this is one of the last things i need to sort out before i can jump ship :)
